.gitlab-ci.yml

```
@@ -144,3 +144,56 @@ validate:
   script:
   - apt-get -y update && apt-get -y install jq
   - ./scripts/validate.sh
+
+validate matrix 1/6:
+   extends: .validate
+   script:
+   - .scripts/matrix/test_case_1.1.sh
```

Shouldn't these all be `./`?

```suggestion
   - ./scripts/matrix/test_case_1.1.sh
```

---

I fixed these and pushed the updated code so it should be ready to be checked again.